packages/@react-spectrum/autocomplete/src/SearchAutocomplete.tsx

```diff
@@ -165,7 +165,7 @@ function _SearchAutocompleteBase<T extends object>(props: SpectrumSearchAutocomp
           isLoading={loadingState === 'loadingMore'}
           onLoadMore={onLoadMore}
           renderEmptyState={() => isAsync && (
-            <span className={classNames(searchAutocompleteStyles, 'no-results')}>
+            <span>
```

Comment:
I think we need this so that `.no-results` is properly styled.

Reply:
Unless those changes are part of SearchAutocomplete fixes to be merged before DocSearch.

Reply:
Moved them here: https://github.com/adobe/react-spectrum/pull/4252